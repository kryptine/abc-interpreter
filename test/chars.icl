module chars

import _SystemArray
import StdClass
import StdInt

Start = ([c \\ c <-: s], [c \\ c <-: s2])
where
	s :: String
	s = {c \\ c <- ['I have only proved it correct.']}

	s2 = "I have only proved it correct"
