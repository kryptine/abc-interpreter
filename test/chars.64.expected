(_Tuple ['I',' ','h','a','v','e',' ','o','n','l','y',' ','p','r','o','v','e','d',' ','i','t',' ','c','o','r','r','e','c','t','.'] ['I',' ','h','a','v','e',' ','o','n','l','y',' ','p','r','o','v','e','d',' ','i','t',' ','c','o','r','r','e','c','t','.'])
halt at 883
919 261225 262144
